### PR TITLE
adding yarn to gitignore

### PR DESCRIPTION
**Pull request by sweetcuptea**
**What issue/feature/bugfix/improvement did you solve?**
---
when running `yarn` on the command line the `.yarn` folder will go into git changes

**How you resolve the issue**
---
add yarn to `.gitignore` so that the `.yarn` folder or other files related to `yarn` are not included in git changes

### CODE REVIEW

**Comment by trihargianto** [comment]
What is your yarn version @sweetcuptea ?

**Comment by trihargianto** [comment]
mine is v1

Screenshot 2023-05-11 at 18 21 02

**Comment by sweetcuptea** [comment]
> mine is v1
> 
> Screenshot 2023-05-11 at 18 21 02

mine is v3 i think
image

**Comment by trihargianto** [comment]
> > mine is v1
> > Screenshot 2023-05-11 at 18 21 02
> 
> mine is v3 i think image

can you use version 1 instead?

**Comment by sweetcuptea** [comment]
can you please explain why should we use version 1 ?

**Comment by trihargianto** [comment]
> can you please explain why should we use version 1 ?

I’m not ready to use the newer version 😅

**Comment by sweetcuptea** [comment]
> I’m not ready to use the newer version 😅

okay, got it

**Comment by trihargianto** [comment]
> > I’m not ready to use the newer version 😅
> 
> okay, got it

as far as I know, .yarn folder wont be exist on version 1

**Comment by sweetcuptea** [comment]
> as far as I know, .yarn folder wont be exist on version 1

yes maybe the problem came from yarn version 3

**Comment by sweetcuptea** [comment]
image

**Comment by trihargianto** [comment]
> image

Ok. Please remove the .yarn folder and run yarn install again

File: .gitignore [hunk 1]
```diff
@@ -46,3 +46,19 @@ lerna-debug.log*
...
+!.yarn/unplugged
+!.yarn/build-state.yml
+!.yarn/install-state.gz
+!.yarn/plugins
+!.yarn/releases
+!.yarn/sdks
+!.yarn/versions
+.pnp.*
+.yarnrc
+yarn.lock
```
Comment (trihargianto) [inline .gitignore:63]:
```suggestion

```

Ok. yarn.lock is required. Please don’t ignore this

Comment (sweetcuptea) [reply .gitignore:63]: got it

Comment (trihargianto) [reply .gitignore:63]: https://yarnpkg.com/getting-started/qa/#which-files-should-be-gitignored

Comment (sweetcuptea) [reply .gitignore:63]: which one that you recommend the most?

Comment (trihargianto) [reply .gitignore:63]: Just follow the documentation

Comment (trihargianto) [reply .gitignore:63]: I dont familiar with it 😅

**Pull request merged**